vspace/src/main/java/edu/asu/diging/vspace/web/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,79 @@
+package edu.asu.diging.vspace.web.exception;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.ModelAndView;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
```

Comment:
I wouldn't call it global, since it's not globally handling exceptions. Just for the browser part.